### PR TITLE
Re-styling library based on TEC design definitions.

### PR DESCRIPTION
**Pull request by famole**
https://www.figma.com/file/b6kx4mkaHxzq4TPzaI1eRY/TEC-Dapp-UI?node-id=198%3A84

### CODE REVIEW

File: src/components/BaseStyles/BaseStyles.js [hunk 1]
```diff
@@ -2,26 +2,35 @@ import React from 'react'
...
-  '400': { url: overpassMonoLightWoff2, format: 'woff2' },
+const BAI_JAMJUREE_FONTS = {
+  '400': { url: baiJamjureeLightWoff2, format: 'woff2' },
+  '600': { url: baiJamjureeRegularWoff2, format: 'woff2' },
+  '800': { url: baiJamjureeSemiBoldWoff2, format: 'woff2' },
+  '900': { url: baiJamjureeBoldWoff2, format: 'woff2' },
 }
 
 function fontSrc(publicUrl, { url, format }) {
+  console.log(`url(${publicUrl + url}) format('${format}')`)
```
Comment (vivianedias) [inline src/components/BaseStyles/BaseStyles.js:33]: heres a console.log!

**Review by vivianedias — CHANGES_REQUESTED** [review]
Great work! Just one small detail 🚀

File: src/components/BaseStyles/BaseStyles.js (hunk 1 above)
Comment (famole) [reply src/components/BaseStyles/BaseStyles.js:33]: ouch! you're right! I'm updating this, thanks vivi!

**Comment by famole** [comment]
This pr it's ready to be closed, the issue on vercel is related tu the sandbox that has been disabled, this is why vercel fails.

**Pull request merged**